writing-apps/code-style.md

```diff
@@ -21,6 +21,12 @@ my_function_name ();
 var my_instance = new Object ();
 ```
 
+An exception is admitted for Gettext-localized strings, where no space should go between the underscore and the opening parenthese:
+```csharp
+download = new Gtk.Button ();
+download.tooltip_text = _("Download");
```

Comment:
```suggestion
// Space before parentheses since it's normal method:
button.label = set_label_string ();
// No space before parentheses since it's gettext-localized string:
button.tooltip_text = _("Download");
```

I suppose a comparison with a normal method like this might be more understandable IMO.